app/javascript/shared/components/MarkdownEditor.css

```diff
@@ -6,15 +6,6 @@
   min-height: 10.125rem;
 }
 
-.markdown-editor__fullscreen-preview-wrapper
-  > .markdown-editor__fullscreen-preview-editor {
+.markdown-editor__fullscreen-preview-wrapper>.markdown-editor__fullscreen-preview-editor {
   @apply max-w-3xl mx-auto;
 }
-
-.markdown-editor__textarea {
-  max-height: 12rem;
-}
-
-.markdown-editor__textarea--full-screen {
-  max-height: 100% !important;
-}
```

Comment:
@indicozy - This file has changes unrelated to this PR. Could you please stash or move them to a new PR?